services/galley/test/integration/API/Teams.hs

```diff
@@ -582,24 +582,47 @@ testUpdateTeam g b c = do
 testUpdateTeamMember :: Galley -> Brig -> Cannon -> Maybe Aws.Env -> Http ()
 testUpdateTeamMember g b c a = do
     owner <- Util.randomUser b
-    let p = Util.symmPermissions [DeleteConversation]
+    let p = Util.symmPermissions [SetMemberPermissions]
     member <- flip newTeamMember p <$> Util.randomUser b
     Util.connectUsers b owner (list1 (member^.userId) [])
     tid <- Util.createTeam g "foo" owner [member]
-    let change = newNewTeamMember (member & permissions .~ fullPermissions)
+    -- Must have at least 1 member with full permissions
+    let changeOwner = newNewTeamMember (newTeamMember owner p)
```

Comment:
Code refers to `owner` in some places (especially tests) but `FullPermissionMember` in others. Can we use consistent naming?